### PR TITLE
docs: add shamefully hoist mention for pnpm users on example launch.json

### PR DESCRIPTION
**Pull request by ThimoDEV**
### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation, readme or JSdoc annotations)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [ ] I have added tests (if possible).
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: docs/2.guide/3.going-further/9.debugging.md [hunk 1]
```diff
@@ -38,6 +38,10 @@ It is possible to debug your Nuxt app in your IDE while you are developing it.
 
 You may need to update the config below with a path to your web browser. For more information, visit the [VS Code documentation about debug configuration](https://go.microsoft.com/fwlink/?linkid=830387).
 
+::important
+Make sure you have shamefully-hoist enabled if you use pnpm to properly start your server in debug mode with below configuration
```
Comment (DamianGlowala) [inline docs/2.guide/3.going-further/9.debugging.md:42]:
```suggestion
Make sure you have shamefully-hoist enabled if you use pnpm to properly start your server in debug mode with below configuration.
```

File: docs/2.guide/3.going-further/9.debugging.md [hunk 2]
```diff
@@ -38,6 +38,10 @@ It is possible to debug your Nuxt app in your IDE while you are developing it.
 
 You may need to update the config below with a path to your web browser. For more information, visit the [VS Code documentation about debug configuration](https://go.microsoft.com/fwlink/?linkid=830387).
 
+::important
+Make sure you have shamefully-hoist enabled if you use pnpm to properly start your server in debug mode with below configuration.
```
Comment (danielroe) [inline docs/2.guide/3.going-further/9.debugging.md:42]: Wouldn't it be better to suggest adding `nuxi` to devDependencies?

Comment (ThimoDEV) [reply docs/2.guide/3.going-further/9.debugging.md:42]: AH yes thats a cleaner solution. I didn't come up with that before. 
Would: 
> _Make sure you have nuxi installed as Devdependency if you use pnpm to properly start your server in debug mode with below configuration._ 

suffice?

Comment (danielroe) [reply docs/2.guide/3.going-further/9.debugging.md:42]: That sounds good!

> If you use `pnpm`, you will need to have `nuxi` installed as a devDependency for the configuration below to work.

Comment (danielroe) [inline docs/2.guide/3.going-further/9.debugging.md:42]:
```suggestion
If you use `pnpm`, you will need to have `nuxi` installed as a devDependency for the configuration below to work.
```

**Review by danielroe — APPROVED** [review]

**Pull request merged**